src/convertToObject.js

```diff
@@ -6,7 +6,17 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
-}
+  const stylesObject = {};
+  const stylesArray = sourceString.split(';');
```

Comment:
The variable 'stylesArray' could be named more descriptively. Since it represents individual style declarations, a name like 'styleDeclarations' might be more informative.